api/RTCRtpEncodingParameters.json

```
@@ -210,7 +210,7 @@
           "spec_url": "https://w3c.github.io/webrtc-extensions/#dom-rtcrtpencodingparameters-maxframerate",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "74"
```

So for the actual change here, it looks like Chrome 81 is the right version for `maxFramerate`:
https://storage.googleapis.com/chromium-find-releases-static/9fe.html#9fe620ffcfaf81dc6ab3838a7d1ac5368fc1c53d

```suggestion
              "version_added": "81"
```

---

I was in a rush for this PR, obviously a few things are wrong. I've updated the commit to reflect the changes.